restcomm/restcomm.commons/src/main/java/org/restcomm/connect/commons/configuration/sets/MainConfigurationSet.java

```diff
@@ -63,4 +63,6 @@
     String getClientAlgorithm();
 
     String getClientQOP();
+
+    String getClearTextPasswordAlgorithm();
```

Comment:
is this name making sense?
i was expecting getDefaultPasswordAlgorithm() or soemthing like that...

Reply:
do we really need to make the token to identify clearText password algortihm to be configurable?

Reply:
@jaimecasero we have `String getClientAlgorithm();` which will return the default client password algorithm which in currently is `md5` and its configurable at `restcomm.xml`

I added the `String getClearTextPasswordAlgorithm();` in order to be able to get the `cleartext`  string from around the project using configuration instead of providing the string every time. In few words, to prevent tedious mistakes such as `cleartext`, `clear_text`, `clearText` etc.